adapters/rgb_adapter.py

```diff
@@ -36,16 +36,23 @@ def handleCommand(self, alias, device, device_data, command, level, color):
             green = colorObject['g']
             red = colorObject['r']
             blue = colorObject['b']
+            ttime = 1
 
-            return {
-                'topic': device_data['friendly_name'] + '/set',
-                'payload': json.dumps({
+            payload = json.dumps({
                     "state": "ON",
+              # Disabled the transition time for now because hue bulb/zigbee2mqtt will
```

Comment:
@JanJaapKo what is the purpose of changing this file: as I see nothing was changed eventually?